### PR TITLE
Ports pixel shifting

### PR DESCRIPTION
**Pull request by bruhlookatthisdood**
## About The Pull Request

Port of Skyrat #870
https://i.imgur.com/0Pn10cV.gif

## Why It's Good For The Game

RP
And humpi-

## Changelog
:cl:
add: Pixel shifting
/:cl:

### CODE REVIEW

**Comment by bruhlookatthisdood** [comment]
why does annie not like

**Comment by SqueezyS** [comment]
Based.

File: code/modules/mob/pixel_shift.dm [hunk 1]
```diff
@@ -0,0 +1,67 @@
+/mob
+	///Whether the mob is pixel shifted or not
+	var/is_shifted
```
Comment (JohnFulpWillard) [inline code/modules/mob/pixel_shift.dm:3]: This should preferably be on /mob instead so it doesn't screw up vscode searching

Comment (bruhlookatthisdood) [reply code/modules/mob/pixel_shift.dm:3]: so var/mob/is_shifted?

Comment (bruhlookatthisdood) [reply code/modules/mob/pixel_shift.dm:3]: sorry am idiot nevermind

**Comment by DaddyIssues98** [comment]
https://github.com/Skyrat-SS13/Skyrat-tg/pull/1512

this help.

**Comment by bruhlookatthisdood** [comment]
> [Skyrat-SS13/Skyrat-tg#1512](https://github.com/Skyrat-SS13/Skyrat-tg/pull/1512)
> 
> this help.

euuhguhghgh fiiiiine
ill do it in like 1 minute - 5 or 6 hours.

File: code/modules/mob/pixel_shift.dm (hunk 1 above)
Comment (bruhlookatthisdood) [reply code/modules/mob/pixel_shift.dm:3]: actually cant find a place to put it.

Comment (JohnFulpWillard) [reply code/modules/mob/pixel_shift.dm]: /code/modules/mob/mob_defines.dm

**Comment by bruhlookatthisdood** [comment]
the code
DOSENT ALIIIIGN!!!

**Comment by bruhlookatthisdood** [comment]
its back, cope.

**Comment by Kitsunemitsu** [comment]
Did anyone even test it!?

**Comment by bruhlookatthisdood** [comment]
i tried to tell them to do it from ooc
no one likes me ):
worked on local though.

**Comment by DaddyIssues98** [comment]
i didnt even knew that was testmerge.

**Comment by Kitsunemitsu** [comment]
It workes, merging

**Pull request merged**